README.md

```diff
@@ -135,6 +135,21 @@ go build -o <path> ./ipinfo/
 
 Replace `<path>` with the required location.
 
+### Note for windows users
```

Comment:
We should create a new section where https://github.com/ipinfo/cli#disabling-color-output currently is, titled "Colored Output", move "Disabling Color Output" under it, and then add this section under it as well.

If you don't have time for this I'll merge this into the `uman/fixes` branch of mine and figure it out before it reaches `master`.

Reply:
Okay

Reply:
I seem to be a bit confused so I think you should just do this the way you want

Reply:
No problem.